schema/manager_test.go

```diff
@@ -96,7 +96,7 @@ func TestMustDefaultConfig(t *testing.T) {
 	if config == nil {
 		t.Error("Expected config to not be empty")
 	}
-	if config.Addresses.Gateway != "/ip4/127.0.0.1/tcp/4002" {
+	if config.Addresses.Gateway[0] != "/ip4/127.0.0.1/tcp/4002" {
```

Comment:
Where did this change from a string to a slice?

Reply:
In go-ipfs.. ipfs-config I believe.

Reply:
I don't see any migration handling this change. Did I miss something?